go/tools/test_init/test_init.go

```diff
@@ -0,0 +1,29 @@
+package test_init
```

Comment:
Need a copyright notice (copy from another file).

Also, please rename to `testinit`. Sorry it's a bit nitpicky: most package are all lowercase, no separators.

Reply:
done